src/docs/platform-architecture-reference/nsx-networking.md

```diff
@@ -0,0 +1,153 @@
+# NSX SDN Cluster <!-- omit in toc -->
+
+- [Segment](#segment)
+- [SNAT IP](#snat-ip)
+- [NetPol differences](#netpol-differences)
+  - [Egress](#egress)
+  - [Route access](#route-access)
+- [Forward Proxy](#forward-proxy)
+- [Data Classification](#data-classification)
+
+## Segment
+
+Your namespace will come with a /24 subnet for your pods. This subnet is private, but routable within the datacenter. If you do not have a SNAT IP, then you can use this subnet in traditional firewall rules when connecting to the classic datacenter.
+
+```console
+# oc get ns steve-test -o yaml | grep subnet
+    ncp/subnet-0: 10.90.89.0/24
+```
+
+## SNAT IP
+
+`*-tools` namespaces have a SNAT IP and it will be listed in the `annotations` of the namespace.
+
+```console
+# oc get ns steve-tools -o yaml | grep snat
+    ncp/snat_ip: 142.34.x.x
+```
+
+The SNAT IP will be used for all traffic leaving your namespace and should be used instead in firewall rules. This is so that build pods can easily connect to the internet and pull down code. Build pods cannot be labeled and so need to have less guardrails applied.
+
+## NetPol differences
+
+### Egress
+
+Egress is now managed by network policy in addition to Ingress. The deny by default policy now includes some rules to allow access to DNS, the internal image registry, and the Kube API.
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: platform-services-controlled-deny-by-default
+  namespace: steve-test
+spec:
+  podSelector: {}
+  egress:
+    - ports:
+        - protocol: TCP
+          port: 5353
+        - protocol: UDP
+          port: 5353
+      to:
+        - podSelector:
+            matchLabels:
+              dns.operator.openshift.io/daemonset-dns: default
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: openshift-dns
+    - ports:
+      - port: 5000
+        protocol: TCP
+      to:
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: openshift-image-registry
+        podSelector:
+          matchLabels:
+            docker-registry: default
+    - ports:
+      - port: 6443
+        protocol: TCP
+      to:
+      - ipBlock:
+          cidr: # IP of master server
```

Comment:
```suggestion
      - ipBlock:
          cidr: # IP of master server
    - ports:
      - port: 8080
        protocol: TCP
      to:
      - ipBlock:
          cidr: # IP of OCIO Forward Web Proxy, include full list         
```

Added forward proxy config for completeness.

Reply:
My understanding is we don't want access to the FP by default. I noted below in the FP section that teams need to add a netpol to allow it if they want internet access. This also lets them limit FP access to just a subset of pods and prevent a DB pod for example from being able to contact the internet,

Reply:
fair enough. :)

Reply:
Added an example netpol for forward proxy